modules/building/pages/rerunning.adoc

```diff
@@ -1,6 +1,6 @@
 = Retriggering build pipelines
 
-Occasionally, your build pipeline may fail unexpectedly, necessitating a retrigger. For pull requests, this can be achieved by adding comments. For post-merge builds, retriggering can be done either through UI, or by using the API to add an annotation to your Component resource.
+Occasionally, your build pipeline may fail unexpectedly, necessitating a retrigger. For pull requests, this can be achieved by adding comments to the PR on GitHub. For post-merge builds, retriggering can be done either through UI, or by using the API to add an annotation to your Component resource.
```

Comment:
We support other source control providers as well. Can we generalize this more? Or should we use terminology like "PR/MR"?

Reply:
How does it sound now?